### PR TITLE
Fix the medium link and update the EOL

### PR DESCRIPTION
**Pull request by wivwiv**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,5 +63,5 @@ You can reach the EMQ community and developers via the following channels.
 - [Twitter](https://twitter.com/EMQTech)
 - [Facebook](https://www.facebook.com/emqxmqtt)
 - [Reddit](https://www.reddit.com/r/emqx/)
-- [Blog](https://medium.com/@emqtt)
+- [Blog](https://medium.com/@emqx)
```
Comment (sigming) [inline README.md]: `https://emqx.medium.com/`

File: en_US/awesome/awesome.md [hunk 2]
```diff
@@ -33,8 +33,8 @@ You can contact the EMQ community and developers through the following channels:
 
   - [EMQX Slack](https://slack-invite.emqx.io)
   - [Twitter](https://twitter.com/EMQTech)
-  - [Forum](https://groups.google.com/d/forum/emqtt)
-  - [Medium](https://medium.com/@emqtt)
+  - [Forum](https://github.com/emqx/emqx/issues)
+  - [Medium](https://medium.com/@emqx)
```
Comment (sigming) [inline en_US/awesome/awesome.md]: `https://emqx.medium.com/`

File: en_US/faq/faq.md [hunk 3]
```diff
@@ -29,7 +29,7 @@ EMQX totally has [3 products.](https://www.emqx.com/en/products/emqx) Different
...
 - EMQX Enterprise: EMQX enterprise version.  It is based on the open source version, and adds data persistence (support Redis, MySQL, MongoDB or PostgreSQL), data bridge to Kafka, LoRaWAN support, EMQX monitoring, Kubernetes deployment etc. It supports 1M level concurrent MQTT connections.
-- EMQX Platform: EMQX Platform version is based on the Enterprise version，and support 10M level concurrent MQTT connections. We provide consulting service for complex IoT platforms, such as cross data center solutions. All kinds of services building an IoT platform can be provided, such as consulting, training, architect design, customized development, platform implementation, testing and operation.
+- EMQX Cloud: [EMQX Cloud](https://www.emqx.com/cloud) is an MQTT middleware for the IoT from EMQ. As the world's first fully managed MQTT 5.0 public cloud service, EMQX Cloud provides a one-stop O&M colocation and a unique isolated environment for MQTT services. In the era of Internet of Everything, EMQX Cloud can help you quickly build industry applications and easily realize the collection, transmission, computation and persistence of IoT data.
```
Comment (sigming) [inline en_US/faq/faq.md:32]: `https://www.emqx.com/en/cloud`

File: zh_CN/awesome/awesome.md [hunk 4]
```diff
@@ -33,8 +33,8 @@ ref:
 
   - [EMQX Slack](https://slack-invite.emqx.io)
   - [Twitter](https://twitter.com/EMQTech)
-  - [Forum](https://groups.google.com/d/forum/emqtt)
-  - [Medium](https://medium.com/@emqtt)
+  - [问答社区](https://askemq.com/c/emqx)
+  - [Medium](https://medium.com/@emqx)
```
Comment (sigming) [inline zh_CN/awesome/awesome.md]: https://emqx.medium.com/

File: zh_CN/faq/faq.md [hunk 5]
```diff
@@ -63,7 +63,7 @@ EMQX 公司主要提供[三个产品](https://www.emqx.com/zh/products/emqx)，
 
 - EMQX Enterprise：EMQX 企业版，在开源版基础上，增加了数据持久化 Redis、MySQL、MongoDB 或 PostgreSQL，数据桥接转发 Kafka，LoRaWAN 支持，监控管理，Kubernates 部署等方面的支持；支持百万级并发连接；
 
-- EMQX Platform：EMQX 平台版，在企业版基础上，支持千万级的连接和跨数据中心的解决方案，提供物联网平台全生命周期中需要的各种服务（咨询、培训、架构设计、定制开发、平台建设、功能测试与运维服务）。
+- EMQX Cloud：[EMQX Cloud](https://www.emqx.com/cloud) 是 EMQ 公司推出的一款面向物联网领域的 MQTT 消息中间件产品。作为全球首个全托管的 MQTT 5.0 公有云服务，EMQX Cloud 提供了一站式运维代管、独有隔离环境的 MQTT 消息服务。在万物互联的时代，EMQX Cloud 可以帮助您快速构建面向物联网领域的行业应用，轻松实现物联网数据的采集、传输、计算和持久化。
```
Comment (sigming) [inline zh_CN/faq/faq.md]: `https://www.emqx.com/zh/cloud`

File: en_US/faq/faq.md (hunk 3 above)
Comment (wivwiv) [reply en_US/faq/faq.md:32]: 默认就是 en，不用带预语言了吧，看起来正式点？

Comment (sigming) [reply en_US/faq/faq.md:32]: > 默认就是 en，不用带预语言了吧，看起来正式点？

会影响 SEO，百度有时候不管 302，跳转前后的 url 都会收录

**Comment by zmstone** [comment]
is this 4.4 specific or affects 4.3 too ?

**Comment by zmstone** [comment]
> is this 4.4 specific or affects 4.3 too ?

if the same change is needed for 4.3, please send you pr to release-4.3 instead, and forward merge to release-4.4

**Pull request closed without merging**